Assets/Scripts/Interactions/IInteractable.cs

```diff
@@ -0,0 +1,4 @@
+﻿public interface IInteractable
+{
+    void Interact(PlayerHandler interactor);
```

Comment:
This should actually take in the ICanInteract interface

Reply:
Yea Put in wrong type for parameters YIKES